requirements/test.txt

```diff
@@ -1,13 +1,9 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
We should be using Python 3.8 virtualenvs for all `make upgrade` jobs to match the server environment.

Reply:
rebased into 002b2ce